torch/csrc/jit/script/schema_type_parser.cpp

```diff
@@ -33,10 +33,14 @@ namespace script {
 TypeAndAlias SchemaTypeParser::parseBaseType() {
   static std::unordered_map<std::string, TypePtr> type_map = {
       {"Generator", GeneratorType::get()},
+      {"Dimname", StringType::get()},
+      {"DimnameList", ListType::create(StringType::get())}, // TODO Remove this type from the schema parser and write these as Dimname[] instead
       {"ScalarType", IntType::get()},
       {"Layout", IntType::get()},
       {"MemoryFormat", IntType::get()},
-      {"QScheme", IntType::get()},
+      {"Storage", IntType::get()},
+      {"QScheme", IntType::get()},  // TODO This type should be removed from the schema parser, it should use the custom class mechanism instead. @jerryzh
```

Comment:
this is actually an Enum, not class. So I guess it's similar to MemoryFormat and we can remove this TODO (unless we want to add enums to our typesystem)